### PR TITLE
Pass responsibility to n.Makefile for compressing JS, hashing assets and tar'ing apps

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "eslint": "^1.10.3",
     "eslint-plugin-react": "^3.16.1",
     "extract-text-webpack-plugin": "^1.0.1",
+    "haikro": "^2.3.2",
```
Comment (ironsidevsquincy) [inline package.json]: do we need this now?

Comment (matthew-andrews) [inline package.json]: `build`+`build-production` is now completely NBT free so yes, for now, want to tidy up the dependencies but will do that in a later version :]

File: Makefile [hunk 2]
```diff
@@ -27,8 +27,6 @@ build: _webpack_setup
 
 build-production: _webpack_setup
```
Comment (matthew-andrews) [inline Makefile]: is the `_webpack_setup` task still needed? @ironsidevsquincy

Comment (ironsidevsquincy) [inline Makefile]: Says the issue's resolved, will check

File: n-makefile.json [hunk 3]
```diff
@@ -4,6 +4,9 @@
       "./public/main.js": "./client/main.js",
       "./public/main.css": "./client/main.scss",
       "./public/ie8.css": "./client/ie8.scss"
-    }
+    },
+    "includes": [
+      "/n-card/"
+    ]
```
Comment (ironsidevsquincy) [inline n-makefile.json]: don't need this after doing a rebase from master

**Comment by ironsidevsquincy** [comment]
:+1:, though had to get the latest n-makefile already, so do a rebase first

**Pull request merged**